conans/client/conan_api.py

```diff
@@ -604,10 +604,14 @@ def config_rm(self, item):
 
     @api_method
     def config_install_list(self):
+        if not os.path.isfile(self.app.cache.config_install_file):
+            raise ConanException("There is no config data. Need to install config first.")
```

Comment:
@memsharded thinking again, maybe it should an empty list and not an error. WDYT ?

Reply:
It totally makes sense.

Reply:
Yes, maybe not an error.

Reply:
Okay, I've updated to an empty list. For `install --remove` will keep that error, because removing is explicit, you should know what you want to remove.